src/main/java/exception/InvalidMoneyAmountException.java

```diff
@@ -0,0 +1,4 @@
+package exception;
+
+public class InvalidMoneyAmountException extends Exception {
```

Comment:
this exception is supposed to end the program (according to comment in Player line 33)